src/build/supportedExtensions.js

```diff
@@ -96,9 +96,9 @@ exports.extensions = {
     { icon: 'image', extensions: ['jpeg', 'jpg', 'gif', 'png', 'bmp', 'ico'] },
     { icon: 'ionic', extensions: ['ionic.project'], filename: true },
     { icon: 'ionic', extensions: ['ionic.config.json'], filename: true },
-    { icon: 'jade', extensions: ['jade', 'pug', 'tag'] },
```

Comment:
Regarding the `tag` extension, I can't find anything to justify its presence in two places.